# msg for receive 2 sensor optocoupler rotary speed sensor
int32 left_opt
int32 right_opt